src/main/java/helper/Storage.java

```diff
@@ -0,0 +1,63 @@
+package helper;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
+import task.Task;
+
+/**
+ * Handles storage
+ */
+public class Storage {
+
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Load the saved tasks (wip)
+     * @return the list of tasks
+     * @throws DukeException
+     */
+    public List<Task> load() throws DukeException {
+        try {
+            File myFile = new File("duke.txt");
```

Comment:
I think it's supposed to be data/duke.txt
You can make the getParentFile first, then the actual file to do this. (or take a look at my Storage class)
And I think you should refrain from hardcoding "duke.txt" multiple times. 1 time should do.